File: DataBinding/Section/IListBinding/C#/IListBinding.DataLayer/DataProvider.cs
using System.Data.SQLite;

namespace ActiveReports.Samples.IListBinding.DataLayer
{
	internal class DataProvider
	{
		/// <summary>
		/// Returns a new connection object for reading the data in the ProductCollection
		/// </summary>
		internal static SQLiteConnection NewConnection
			=> new SQLiteConnection(ActiveReports.Samples.IListBinding.DataLayer.Properties.Resources.ConnectionString);
	}
}

File: DataBinding/Section/IListBinding/VB.NET/IListBinding.DataLayer/DataProvider.vb
Imports System.Data.SQLite

Friend Class DataProvider

	'<summary>
	' Returns a new connection object for reading the data in the ProductCollection
	'</summary>
	Friend Shared ReadOnly Property NewConnection() As SQLiteConnection
		Get
			Dim connectString As String = My.Resources.ConnectionString
			Return New SQLiteConnection(connectString)
		End Get
	End Property

End Class 'DataProvider
